### PR TITLE
Remove things not needed

### PR DESCRIPTION
**Pull request by Jerrydotpy**
## Description

Removing things that aren't needed

## Checklist

- [ ] - All code changes have been tested, if any
- [ ] - The documentation has been updated accordingly for any code changes
- [ ] - This PR fixes an already existing issue
- [ ] - This PR adds a new feature (new functions, methods, classes, parameters, etc.)
- [ ] - This PR fixes a bug
- [ ] - This PR is a breaking change (removes/renames functions, parameters etc.)
- [x] - This PR is NOT a code change (changes Documentation files, README.md, CONTRIBUTING.md etc.)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,3 +0,0 @@
-[build-system]
```
Comment (TheUntraceable) [inline pyproject.toml:1]: iirc this is needed, *could* be wrong

Comment (Jerrydotpy) [reply pyproject.toml:1]: Well we could regenerate a better one

Comment (nav-github01001) [reply pyproject.toml:1]: This is needed

**Review by nav-github01001 — CHANGES_REQUESTED** [review]
pyproject.toml is the best example i can find in the setuptools docs! without it the library couldn't
be installed

**Pull request closed without merging**